CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 0.2.0
+
+* Add pagination functionality.
+* Add after-action summary and done message to log output.
+* Add check to determine if Fargate task is already running.
+* Add additional unit tests.
+* Update `sendCommand` functionality.
```

Comment:
What does this mean?

Reply:
I may actually remove it since its not really feature related like some of the other callouts you've made.